### PR TITLE
End of year retrospective 2024

### PR DESCRIPTION
**Pull request by choldgraf**
This adds several new blog posts that provide links for some of the bigger projects we've undertaken that we haven't yet spoken about. It also adds an "end of year recap" to cover our major impact for the year. My goal will be to send e-mails to major stakeholders within 2i2c that point to the EOY recap for those that want to learn more.

👀 preview: https://deploy-preview-342--cocky-booth-e7ed17.netlify.app/

Here are the posts I've added:

1. Short post announcing CZI funding back-dated to October
2. Short post announcing Navigation Fund funding
3. Short post about the frx data challenge repo
4. Annual retrospective / impact post

### Ways to provide input / help

- Give these posts a read for general understandability, conciseness, and correctness.
- Make sure I haven't missed any major team impact or effort in the recap post.
- Make sure that we aren't missing any other key blog posts that describe impact we've had this year.
- Generally provide any suggestions for how we can more effectively communicate our impact and major updates over the last year

### CODE REVIEW

File: content/blog/2024/funding-czi/index.md
Comment (aprilmj) [inline content/blog/2024/funding-czi/index.md]: Caption looks copied from the NF post, just needs to be updated to something appropriate to CZI

File: content/blog/2024/frx/index.md
Comment (aprilmj) [inline content/blog/2024/frx/index.md]: > enable their users to 
(missing preposition I think)

File: content/blog/2024/frx/index.md
Comment (aprilmj) [inline content/blog/2024/frx/index.md]: this displays an extra space between the link and the comma that follows it
> [Cellmap Challenge](https://cellmapchallenge.janelia.org/)

File: content/blog/2024/funding-czi/index.md
Comment (aprilmj) [inline content/blog/2024/funding-czi/index.md]: We used the same adverb (incredibly) here and in the NF article. Maybe very grateful or deeply grateful?

File: content/blog/2024/funding-navigation/index.md
Comment (aprilmj) [inline content/blog/2024/funding-navigation/index.md:3]: Backdate this to late November? We got the thumbs up at the end of October.

File: content/blog/2024/retrospective/index.md
Comment (aprilmj) [inline content/blog/2024/retrospective/index.md]: > that I now call the "$1M to $2M budget jump"
I don't have a suggested rewording for this, it just flows weirdly with the next paragraph (feel free to ignore this feedback)

**Review by aprilmj — CHANGES_REQUESTED** [review]
You pulled the impact stories I would have chosen for the year, so no notes there. 

A couple of minor grammatical errors that I corrected and one image caption that needs to be fixed (in funding-czi)

File: content/blog/2024/frx/index.md
Comment (choldgraf) [reply content/blog/2024/frx/index.md]: Oh nice this uncovered a bug in our link rendering in general that is now fixed!

File: content/blog/2024/funding-navigation/index.md
Comment (choldgraf) [reply content/blog/2024/funding-navigation/index.md:3]: I think for this one I'll leave it to now in case Karthik wants to coordinate a release now. It's OK because it's still pretty close in time.

**Comment by choldgraf** [comment]
I've addressed all of @aprilmj's changes, thanks very much for those! Do you think it's safe to post?

I'll hold off until tomorrow or early Wednesday before posting, and would value suggestions in the meantime from anybody else as well.

File: content/blog/2024/funding-czi/index.md
Comment (sgibson91) [inline content/blog/2024/funding-czi/index.md]: I think having "support" twice in this title is a bit redundant. I would perhaps reword.

File: content/blog/2024/frx/index.md
Comment (GeorgianaElena) [inline content/blog/2024/frx/index.md]: Small typo:

```suggestion
  caption="An example leaderboard for a data challenge, taken from the [Cellmap Challenge](https://cellmapchallenge.janelia.org/). Users make submissions that are run against secure and private infrastructure and data, and provides feedback about the submission's performance. Learn more about the FRX challenges project here: https://2i2c.org/frx-challenges/"
```

File: content/blog/2024/frx/index.md
Comment (jnywong) [inline content/blog/2024/frx/index.md]:
```suggestion
title: "`frx-challenges`: A new tool to host data challenges for Frictionless Research Exchanges"
```

File: content/blog/2024/funding-czi/index.md
Comment (jnywong) [reply content/blog/2024/funding-czi/index.md]:
```suggestion
title: Support from CZI to realize 2i2c's mission to help communities create and share knowledge with open infrastructure
```

Thesaurus.com – Strongest matches

accomplish,
achieve,
attain,
get,
produce,
reach,
receive

😄

File: content/blog/2024/retrospective/index.md
Comment (jnywong) [inline content/blog/2024/retrospective/index.md:15]:
```suggestion
In 2024, 2i2c reached the point in an organization's lifecycle when a team has grown enough in size and complexity that you must change your operational style. The informal ways that worked well as a small group don't suffice anymore, and you have to put more effort into aligning and coordinating everyone to ensure you have the same impact.
```

Small style suggestion to avoid repetition.

File: content/blog/2024/retrospective/index.md
Comment (jnywong) [inline content/blog/2024/retrospective/index.md]:
```suggestion
- **We've re-organized our team into separate product and business development teams**, in order to focus on _providing an excellent technical platform and a collection of services that maximizes community impact_, as well as _sustaining this service for our communities_.
```

File: content/blog/2024/retrospective/index.md
Comment (jnywong) [inline content/blog/2024/retrospective/index.md]:
```suggestion
Along the way, we also generated a lot of impact through our collaborations with communities in our network, and in the upstream projects that we support. For more details about our impact, see the summaries below.
```

File: content/blog/2024/retrospective/index.md
Comment (jnywong) [inline content/blog/2024/retrospective/index.md]:
```suggestion
First, we've grown our network of hubs and users through several new partnerships. We **grew the number of active hubs from ~75 to ~105**, and **grew the number of Monthly Active Users (MAUs) from ~8100 to ~9100**.
```

File: content/blog/2024/retrospective/index.md
Comment (jnywong) [inline content/blog/2024/retrospective/index.md]:
```suggestion
Beyond the numbers, we also re-focused our team on reporting impact stories from our collaborations with community members, and have published these into a (growing) list of posts on our blog:
```

Minor style suggestion, take it or leave it :)

File: content/blog/2024/retrospective/index.md
Comment (jnywong) [inline content/blog/2024/retrospective/index.md]:
```suggestion
- We served around 20 communities from Latin America and Africa for the Catalyst project: https://2i2c.org/blog/2024/catalyst-partner-highlights/
```

File: content/blog/2024/retrospective/index.md
Comment (jnywong) [inline content/blog/2024/retrospective/index.md]:
```suggestion
- We ran a pilot for an HHMI-funded Spyglass project for reproducing their pre-print with a live interactive environment using BinderHub to support publishing infrastructure: https://2i2c.org/blog/2024/hhmi-spyglass-mysql/
```

File: content/blog/2024/retrospective/index.md
Comment (jnywong) [inline content/blog/2024/retrospective/index.md]:
```suggestion
- We built `frx-challenges`, a tool to help communities host data challenges with secure, automated evaluation of submissions.
```

File: content/blog/2024/retrospective/index.md
Comment (jnywong) [inline content/blog/2024/retrospective/index.md]:
```suggestion
2025 is going to be a critical year for 2i2c to build upon the work we began this year and achieve a more sustainable and scalable community model. Here are the main areas that will guide our work in the new year and into 2026, pulled from our [recent proposal from The Navigation Fund](/blog/2024/funding-navigation/index.md):
```

**Review by jnywong — APPROVED** [review]
🚀 🚀 🚀 🚀 

2️⃣ 0️⃣ 2️⃣ 4️⃣

**Comment by choldgraf** [comment]
I've addressed everybody's comments above, (those were really helpful, thanks y'all ❤️).

I've also added a new blog post (because why not lol) that shares the content @colliand wrote up in https://github.com/2i2c-org/meta/issues/1691

I've got approval from The Navigation Fund that the snippet looks good. I haven't yet heard back from CZI. If they don't get back by today, I might just remove that one and mention it informally.

I think that we are good to go?

File: content/blog/2024/frx/index.md
Comment (yuvipanda) [inline content/blog/2024/frx/index.md]: I'm not sure what org Kristen Ratan belongs to, but let's add them to the acknowledgement as well.

File: content/blog/2024/frx/index.md
Comment (yuvipanda) [inline content/blog/2024/frx/index.md]:
```suggestion
- **Evaluations must be totally secure**, so that submissions can't do nefarious things like mine cryptocurrency or exfiltrate the challenge's private data.
```

Comment (yuvipanda) [reply content/blog/2024/frx/index.md]: Since the submission must actually have access to the challenge's private data, but not actually be able to take the data itself back

File: content/blog/2024/frx/index.md
Comment (yuvipanda) [inline content/blog/2024/frx/index.md]: I would suggest not putting this so prominently, as it doesn't actually have any useful info right now on how to use it.

**Comment by colliand** [comment]
I edited the comment in #1691 to make things prettier.

**Comment by choldgraf** [comment]
OK I'm merging this now that we've had a few rounds of iteration! We can let it marinate overnight and we can change stuff tomorrow morning if anything comes to mind. @aprilmj feel free to link to the AGU post once it's up. And let's coordinate on the right time to "release" the end of year post.

**Comment by aprilmj** [comment]
When you say "release" the blog post, @choldgraf I assume you mean social media? Since the post itself is published. 

I'd suggest we share it early next week, so people have time to share if they'd like before the 2i2c year officially ends.

**Comment by choldgraf** [comment]
Yeah by "release" I mean "intentionally signal-boost it". e.g., social media, but also something like e-mail campaigns etc

**Pull request merged**
